qdrant/v1.1.x/integrations.md

```diff
@@ -6,6 +6,67 @@ Qdrant is a vector database performing an approximate nearest neighbours search
 as a standalone system, yet, in some cases, you may find it easier to implement your semantic search application using some
 higher-level libraries. Some of such projects provide ready-to-go integrations and here is a curated list of them.
 
+## LangChain
+
+LangChain is a library that makes developing Large Language Models based applications much easier. It unifies the interfaces 
+to different libraries, including major embedding providers and Qdrant. Using LangChain, you can focus on the business value 
+instead of writing the boilerplate.
+
+Langchain comes with the Qdrant integration by default. It might be installed with pip:
+
+```bash
+pip install langchain
+```
+
+Qdrant acts as a vector index that may store the embeddings with the documents used to generate them. There are various ways 
+how to us it, but calling `Qdrant.from_texts` is probably the most straightforward way how to get started:
+
+```python
+from langchain.vectorstores import Qdrant
+from langchain.embeddings import HuggingFaceEmbeddings
+
+embeddings = HuggingFaceEmbeddings(
+    model_name="sentence-transformers/all-mpnet-base-v2"
+)
+doc_store = Qdrant.from_texts(
+    texts, embeddings, host=QDRANT_HOST, api_key=QDRANT_API_KEY, collection_name="texts"
+)
+```
+
+If you'd like to know more about running Qdrant in a LangChain-based application, please read our article 
+[Question Answering with LangChain and Qdrant without boilerplate](/articles/langchain-integration/). Some more information
+might also be found in the [LangChain documentation](https://python.langchain.com/en/latest/modules/indexes/vectorstores/examples/qdrant.html).
+
+## LlamaIndex (GPT Index)
+
+LlamaIndex (formerly GPT Index) acts as an interface between your external data and Large Language Models. So you can bring your 
+private data and augment LLMs with it. LlamaIndex simplifies data ingestion and indexing, integrating Qdrant as a vector index.
+
+Installing LlamaIndex is straightforward if we use pip as a package manager:
+
+```bash
+pip install llama-index
+```
+
+LlamaIndex requires providing an instance of `QdrantClient`, so it can interact with Qdrant server.
+
+```python
+from llama_index import GPTQdrantIndex
+
+import qdrant_client
+
+client = qdrant_client.QdrantClient(
+    url="<qdrant-url>",
```

Comment:
it might be better not explicitly put url= here, as it is always confusing. Let's just keep url a first param and let qdrant resolve it internally

Reply:
Done. The url param was removed